components/contribution-flow/ContributionFlowStepsProgress.js

```diff
@@ -70,7 +77,7 @@ const StepInfo = ({ step, stepProfile, stepDetails, stepPayment, stepSummary, is
     if (isFreeTier && getTotalAmount(stepDetails, stepSummary) === 0) {
       return <FormattedMessage id="noPaymentRequired" defaultMessage="No payment required" />;
     } else {
-      return (stepPayment?.paymentMethod && getPaymentMethodName(stepPayment.paymentMethod)) || null;
+      return (!isCrypto && stepPayment?.paymentMethod && getPaymentMethodName(stepPayment.paymentMethod)) || null;
```

Comment:
Why is this necessary? It seems fine if stepPayment?.paymentMethod is null.

Reply:
@znarf : If this condition wasn't there we will see the payment method below the section step; 

![image](https://user-images.githubusercontent.com/12435965/128098237-d6f8fa15-6578-42e0-96e4-26a047f5411e.png)

We don't want this for crypto as per the design. As there's no payment method to choose from such as visa or bank account. This is why I added this one.

Reply:
If `stepPayment?.paymentMethod` is null or undefined, that should not be necessary.

If it is defined, what's its value?

Reply:
Yes, this condition is no longer needed when we add the other parts (the QR code part and stuff). I've removed this at the other PR; https://github.com/opencollective/opencollective-frontend/pull/6725. 👍🏽